.github/workflows/docker.yml

```diff
@@ -0,0 +1,44 @@
+name: Container Image
+
+on:
+  push:
+    branches: [ main ]
+    tags: [ 'v*' ]
+  pull_request:
+    branches: [ main ]
+
+env:
+  IMAGE_NAME: ${{ github.repository }}
+  REGISTRY: ghcr.io
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Log into registry ${{ env.REGISTRY }}
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Extract Docker metadata
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v2
+        with:
+          push: ${{ github.event_name != 'pull_request' }}
```

Comment:
Notably, this *won't* automatically make latest == develop, which is what we did previously. I don't know if we have a set standard across our other bridges.

Reply:
(matrix-hookshot *does* push `latest` when things hit `main`)

Reply:
I'd like us to automate the update of `latest` as well, though it may follow in a separate PR.

Reply:
Believe this will do the right thing, albeit it will publish `develop` as `develop` and `latest`/`v.X.X` for versioned releases. I'm quite comfortable with that, anyway.